hever/sass/style-child-theme-editor.scss

```diff
@@ -8,6 +8,15 @@
  */
 @import "../../varia/sass/abstracts/imports";
 
+/**
+ * Global variables
+ */
+ @import "global-variables";
+
+:root, body {
```

Comment:
Is it still required by Gutenberg to target the `:root` and `body` to get variables to register in the editor?

Reply:
yes, they don't work unless we do it like that :( If we only target `:root `then it doesn't work on the editor, if we target both on both stylesheets, then the customizer doesn't work.